src/Server/WishList.fs

```diff
@@ -2,35 +2,37 @@
 module ServerCode.WishList
 
 open Giraffe
-open Giraffe.Tasks
 open RequestErrors
 open ServerErrors
 open ServerCode.Domain
 open Microsoft.AspNetCore.Http
 open Microsoft.Extensions.Logging
 open System.Threading.Tasks
+open System.Security.Claims
 
 /// Handle the GET on /api/wishlist
 let getWishList (getWishListFromDB: string -> Task<WishList>) next (ctx: HttpContext) =
-    Auth.useToken next ctx (fun token -> task {
```

Comment:
@Krzysztof-Cieslak From an educational point of view, what does the `Auth.useToken` get replaced with in Saturn?

Reply:
Checking if the request is authenticated is done by `pipe_through jwtAuthentication` in `secured` router in WebServer.fs file.

As I've mentioned in the earlier comment this is using ASP.NET functionality and `Challenge` functionality which checks if the request is authorized and provides appropriate, standard responses if the request is not authorized. If the request is authorized we just go to the handler for a particular path (`getWishlist` for example) , and here we can use `ctx.User` to get information about current user - this field is automatically populated by ASP.NET

Reply:
Again, it's just side effect of Saturn using ASP.NET mechanisms for authentication helpers.

Same would happen if we moved to ASP.NET authentication without using Saturn, and just Giraffe's challenge and requiresAuthentication handlers